### ISSUE TITLE
Isn't ssr supported in next14

### ISSUE THREAD

**Issue by OWOfadeAway** [imzbf/md-editor-rt#135]
### Describe the issue

Isn't ssr supported in next14 , Reminds me that I need to write use client，I only used MdPreview and MdCatalog。
![image](https://github.com/imzbf/md-editor-rt/assets/30753898/4e12aa47-e993-4b06-92ab-626cbc84b97c)
![image](https://github.com/imzbf/md-editor-rt/assets/30753898/97bc0e9d-84ae-4340-83d6-937a9f3667bd)

![image](https://github.com/imzbf/md-editor-rt/assets/30753898/faffa984-5292-4adf-9aac-4cb016f1ef30)

### Procedure version

next@14 md-editor-rt@4.11.2

**Comment by imzbf** [comment]
This component is not a server component in nextjs. So you need to write `use client` in this component.

Server side rendering is still available.

**Comment by OWOfadeAway** [comment]
Will the current component perform server-side rendering when I use 'use client'?

**Comment by OWOfadeAway** [comment]
![image](https://github.com/imzbf/md-editor-rt/assets/30753898/052cc874-5dd0-4b31-b19d-7a2d7be9b26d)
and I don't think I used any client components

**Comment by imzbf** [comment]
> Will the current component perform server-side rendering when I use 'use client'?

Yes, to verify it, you only need to use Postman to access your page

**Comment by OWOfadeAway** [comment]
I know the difference，MdPreview and MdCatalog aren't server components？

**Comment by imzbf** [comment]
> I know the difference，MdPreview and MdCatalog aren't server components？

Correct, the editor contains a substantial amount of state and interactions, making it unsuitable for design as a server component.

**Comment by OWOfadeAway** [comment]
tk ! @imzbf
